openapi/components/schemas/GatewayAccountConfig/Adyen.yaml

```
@@ -73,6 +73,10 @@ allOf:
             format: double
             description: Total tax amount as a decimal to send with level 2 enhanced scheme data.
             example: 0.06
+          enableMoto:
+            type: boolean
+            description: Specifies whether to use Mail Order Telephone Order (MOTO).
```

```suggestion
            description: Specifies whether to use Mail Order/Telephone Order (MOTO).
```

On Adyen site, it is written with forward slash

> Mail Order/Telephone Order (MOTO) transaction is a [card-not-present transaction](https://docs.adyen.com/get-started-with-adyen/payment-glossary#card-not-present-cnp) where the shopper provides you with their order and payment details by regular mail (not email), fax, or telephone.